src/runtime/node/crypto/index.ts

```diff
@@ -1,8 +1,17 @@
 // https://nodejs.org/api/crypto.html
 // https://github.com/unjs/uncrypto
-import type nodeCrypto from "node:crypto";
+import type nodeCrypto from "node:crypto"
+import stream from "node:stream";
+// @ts-ignore
+import cryptoBrowserify from "crypto-browserify"
+// @ts-ignore
+import { webTimingSafeEqual } from "@advena/web-timing-safe-equal"
```

Comment:
we should use the browserify version instead of this one https://www.npmjs.com/package/timing-safe-equal